### ISSUE TITLE
Nier:Automata - 6O operator portrait (524220)

### ISSUE THREAD

**Issue by DigitalDevilSummoner** [ValveSoftware/Proton#1543]
When talking to operator 6O, sometimes her portrait will show up as a a bright circle of light with a black square in the middle
![20180920220017_1](https://user-images.githubusercontent.com/43498489/45918639-f8c6a400-be57-11e8-86c7-a0a27e3c6f31.jpg) 
https://gist.github.com/DigitalDevilSummoner/d7a227765539daee04f9fd1d98d2be93

**Comment by kisak-valve** [comment]
Hello @DigitalDevilSummoner, please copy your system information from steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by rkfg-bc** [comment]
I have the same issue although for me this circle also flashes quickly, it happens for Pascal as well so I think it's an issue with all "holographic" portraits. I'm afraid epileptics could even get a seizure when this happens, I'm not affected but this blinking is very uncomfortable. [Here's my system info](https://gist.github.com/rkfg-bc/576c7e85af8cc0ba2324c9347fb748b2). This happened on driver version 396.54 as well as on the newest 410.57. Proton's version is 3.7-6 Beta. Changing the game settings to low or high doesn't affect the issue. I managed to capture two frames where the circle appears and disappears, can't really tell the blinking rate.
![20180923035254_1](https://user-images.githubusercontent.com/43507329/45923125-0f1b4100-bee6-11e8-9d50-483f76fac88d.jpg)
![20180923035300_1](https://user-images.githubusercontent.com/43507329/45923124-0e82aa80-bee6-11e8-917c-b15c2c6d6749.jpg)

**Comment by DuloT34** [comment]
Same issue here, reproduced on Ubuntu 18.04 with NVIDIA drivers 396.54.05, 396.54.06, 410.57

**Comment by Plagman** [comment]
I can see the same thing on all floating portraits during 9S's opening, on a regular non-Pascal Titan X with 396.54.05.

**Comment by rkfg-bc** [comment]
So I tried to record a video of this bug and of course it's not happening anymore (who doesn't hate heisenbugs, raise your hands; no one? thought so). I have two displays, the main is 1920x1200 and the TV (connected via HDMI) is 1920x1080. So when I launched the game on the main display the bug didn't happen and when I switched to TV it's still not happening. The only things that differ is that on TV I launched it in Big Picture and different resolution (I changed it manually in the game settings). Another possibility is using suspend/resume, I'll check if it will happen tomorrow after resuming the system.

**Comment by DigitalDevilSummoner** [comment]
So I tried to see if I could catch the bug yesterday, and aside from the regular crashes, I've had no luck.

**Comment by rkfg** [comment]
So far, the bug seemingly disappeared. I completed my first playthrough and am already in the machine village on the second one. Quite weird as I didn't notice any updates to Proton and the game itself.

**Comment by DigitalDevilSummoner** [comment]
Yeah, from what i've read, the PC version won't be getting any updates any time soon which sucks like hell.

**Comment by recke96** [comment]
Same issue here:
![nier_screenerror](https://user-images.githubusercontent.com/7427710/46316573-d7408780-c5d0-11e8-89ee-01f33a0c672f.jpg)
Here is my system information https://gist.github.com/recke96/187d2bf3f56f4dd3d32a4bea238483b5

**Comment by rkfg** [comment]
Happened to me yesterday again. A simple game restart "fixes" the bug until some next time. It certainly doesn't happen each time, I didn't have it for days.

**Comment by ryncsn** [comment]
Same issue here, I'm running Fedora Rawhide, Proton 3.16-4 Beta, GPU is GTX 1070, nvidia driver version 410.73, mesa 18.2, kernel 4.19.0. Running in 1920x1080, using highest graphic setting.

**Comment by DanMan** [comment]
I'm still having this issue on Fedora 33 (Kernel 5.14.9) with Proton 6.3-7 and Nvidia 470.74. Funny thing is that it wasn't like that from the start, but started happening a few hours into the game. At one time I also had a corrupted start screen after playing for hours when quitting the game, with everything black except for a triangular area in the top right corner. Maybe there's some asset or memory corruption happening?

Sysinfo (partially outdated): https://gist.github.com/DanMan/df94fe2dc319aad76fc097c2f90e6329

**Comment by Wedge009** [comment]
Seems specific to NV GPUs? I didn't observe anything like this having only just played and finished this game in September 2025, amdgpu + vanilla Proton 9.0-4.
